scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigtable/BigTableIO.scala

```diff
@@ -89,6 +84,7 @@ object BigtableRead {
   object ReadParam {
     val DefaultKeyRanges: Seq[ByteKeyRange] = Seq.empty[ByteKeyRange]
     val DefaultRowFilter: RowFilter = null
+    val DefaultMaxBufferElementCount: Option[Int] = None
```

Comment:
As this setting is nullable, I prefer the scala option instead of the `Integer = null`. WDYT ?